geopandas/io/sql.py

```diff
@@ -92,7 +95,35 @@ def load_geom_text(x):
             srid = shapely.get_srid(geoms.iat[0])
             # if no defined SRID in geodatabase, returns SRID of 0
             if srid != 0:
-                crs = "epsg:{}".format(srid)
+                try:
+                    spatial_ref_sys_df = _get_spatial_ref_sys_df(con, srid)
+
+                    spatial_ref_sys_sql = (
+                        f"SELECT srid, auth_name FROM "
+                        f"spatial_ref_sys WHERE srid = {srid}"
+                    )
+                    spatial_ref_sys_df = pd.read_sql(spatial_ref_sys_sql, con)
```

Comment:
```suggestion
```
this bit looks duplicated right now

Reply:
whoops, what am embarrassing thing to miss, thanks!